app/relisten/(tabs)/artists/[artistUuid]/venues.tsx

```diff
@@ -1,16 +1,104 @@
+import React from 'react';
+import Flex from '@/relisten/components/flex';
+import { RefreshContextProvider } from '@/relisten/components/refresh_context';
+import { SubtitleRow, SubtitleText } from '@/relisten/components/row_subtitle';
+import RowTitle from '@/relisten/components/row_title';
+import { DisappearingHeaderScreen } from '@/relisten/components/screens/disappearing_title_screen';
+import { SectionedListItem } from '@/relisten/components/sectioned_list_item';
+import { Venue } from '@/relisten/realm/models/venue';
 import { useArtistVenues } from '@/relisten/realm/models/venue_repo';
 import { useLocalSearchParams } from 'expo-router';
-import { Text, View } from 'react-native';
+import {
+  FilterableList,
+  FilterableListProps,
+} from '@/relisten/components/filtering/filterable_list';
+import { RelistenText } from '@/relisten/components/relisten_text';
+import Plur from '@/relisten/components/plur';
+import { FilteringProvider, Filter, SortDirection } from '@/relisten/components/filtering/filters';
 
 export default function Page() {
-  const { uuid } = useLocalSearchParams();
-
-  const results = useArtistVenues(String(uuid));
+  const { artistUuid } = useLocalSearchParams();
+  const results = useArtistVenues(String(artistUuid));
   const { data } = results;
 
   return (
-    <View style={{ flex: 1, width: '100%' }}>
-      <Text className="text-white">Venue List</Text>
-    </View>
+    <RefreshContextProvider networkBackedResults={results}>
+      <DisappearingHeaderScreen
+        headerHeight={50}
```

Comment:
we should replace all of these `headerHeight={50}` with [`useHeaderHeight()`](https://reactnavigation.org/docs/elements/#useheaderheight)

Reply:
Updated.